x/protocolpool/types/keys.go

```diff
@@ -0,0 +1,29 @@
+package types
+
+import (
+	"cosmossdk.io/collections"
+)
+
+const (
+	// ModuleName is the module name constant used in many places
+	//
+	// The module account associated with this name is the x/protocolpool community pool module account.
+	// It can receive funds from distribution from users who use FundCommunityPool or from the ProtocolPoolDistrAccount.
+	ModuleName = "protocolpool"
+
+	// ProtocolPoolDistrAccount is an intermediary account that holds the funds to be distributed to the protocolpool accounts.
+	//
+	// It can receive funds from `x/distribution` and distribute them to continuous funds and the community pool.
+	ProtocolPoolDistrAccount = "protocolpool_distr"
```

Comment:
im still not sure i understand the difference between these two accounts based on these comments

when is protoclpool funded? when is protocolpool_distr funded?

Reply:
updated docs with "Funded By" and "Distributes To"

Reply:
also renamed to `ProtocolPoolEscrowAccount` because it is an escrow that temporarily holds funds until it either distributes towards 
- Continuous Funds
- CommunityPool